packages/functionals/botpress-qna/README.md

```diff
@@ -77,6 +77,16 @@ const questionsFlatExported = await bp.qna.export({ flat: true })
 // ]
 ```
 
+## Controlling whether Q&A should intercept
```

Comment:
IMO better wording: `if` instead of `whether`

Reply:
@emirotin , fixed that.